entrypoint.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+
+export PATH=/usr/bin:/bin:/usr/sbin:/sbin
+
+tuser=tsvc
+name=telemetry-server
+_localstatedir=/var
+_tsvcdir=${_localstatedir}/lib/${tuser}
+_certdir=${_tsvcdir}/certs
+
+[ -d ${_certdir} ] && cp -a ${_certdir}/* /etc/pki/trust/anchors
+
+update-ca-certificates
+
+#For now commenting this out until we have a telemetry-server package built
+#if [[ ! -d ${_tsvcdir} ]]; then
+#    echo "Error: no ${_tsvcdir} directory found; did you bind mount the service account dir to ${_tsvcdir}?"
+#    exit 1
+#fi
+
```

Comment:
May want to repeat the chown to the service account here to ensure that the telemetry user can access files on a mounted volume, e.g. if the service account UID changes over time.

Reply:
Updated

Reply:
This will do for now... May need to revisit in the long run as there may be better ways of handling some of this with docker options. ;-)